ddp-workspace/projects/ddp-dsm-ui/src/app/FON/pages/patients-list/patients-list.component.ts

```diff
@@ -44,9 +44,7 @@ export class PatientsListComponent implements OnInit {
     this.loading$ = this.storeService.getParticipantsLoadingStatus;
 
     const qParams = JSON.parse(localStorage.getItem(this.LSParams));
-
-    localStorage.setItem(this.LSParams,
-      JSON.stringify({from: qParams?.from || 0, to: qParams?.to || 10}));
+    this.setToLocalStorage({from: qParams?.from || 0, to: qParams?.to || 10});
```

Comment:
**_#minor_**

`{from: qParams?.from || 0, to: qParams?.to || 10}` is used twice (line 47, 50). 
Can be extracted as a constant.

Reply:
Also suggest keeping default values as constants 
like 
```
DEFAULT_FROM_VALUE = 0;
DEFAULT_TO_VALUE = 10;
```

Reply:
I agree, I was just in a rush when I wrote that. This component requires refactoring. But for now, I will change as you suggested. Anyway, later I will get back to it.

Reply:
Done